test_rclcpp/test/node_check_names.cpp

```diff
@@ -12,7 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <algorithm>
```

Comment:
It doesn't look like `<algorithm>` is being used in this file.

Reply:
Must have forgotten to remove it...